engine/flink/process/src/test/scala/pl/touk/nussknacker/engine/process/runner/FlinkTestMainSpec.scala

```diff
@@ -309,7 +309,8 @@ class FlinkTestMainSpec extends FunSuite with Matchers with Inside with BeforeAn
     SinkForInts.data should have length 0
   }
 
-  test("be able to test process with signals") {
+  //TODO: remove it
+  ignore("be able to test process with signals") {
```

Comment:
Are you sure that is okey? Why you comment this test?

Reply:
:+1: I moved test to kafka module